lib/calculateUtils.js

```diff
@@ -84,6 +75,22 @@ export function calcGridItemPosition(
   else {
     out.top = Math.round((rowHeight + margin[1]) * y + containerPadding[1]);
     out.left = Math.round((colWidth + margin[0]) * x + containerPadding[0]);
+    right = Math.round((colWidth + margin[0]) * (x + w) + containerPadding[0] - margin[0]);
+    bottom = Math.round((rowHeight + margin[1]) * (y + h) + containerPadding[1] - margin[1]);
+  }
+
+  // If resizing, use the exact width and height as returned from resizing callbacks.
+  if (state && state.resizing) {
+    out.width = Math.round(state.resizing.width);
+    out.height = Math.round(state.resizing.height);
+  } else if (right > 0 && bottom > 0) {
+    out.width = right - out.left;
+    out.height = bottom - out.top;
+  }
```

Comment:
Are we always going to have right and bottom > 0 at the same time?  
Would we want to make them independent?
e.g.
```suggestion
  } else  {
     if (right > 0) {
         out.width = right - out.left;     
      }
      if( bottom > 0) {
          out.height = bottom - out.top;
      }
  }
```
```

Reply:
Seems like it covers all cases that matter - we were using this same code in this PR in the old package for this same fix so I just kept it as-is.

Reply:
In that case, carry on.